server.rb

```diff
@@ -56,7 +63,7 @@ def get_message
   post "/suggest_song" do
     # enter Artist, Title, Album=nil
     # submit and save to Songs table
-    # require_current_user
+    require_current_user
```

Comment:
Okay. Can you go ahead and make and push up that change? While you're at it, go ahead and add a similar call anywhere else in the app that needs it.